Failing mission-doc build documentation workflow

**Checklist (Please check before submitting)**

* [x] I reviewed the [Contributing Guide](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md).
* [x] I performed a cursory search to see if the bug report is relevant, not redundant, nor in conflict with other tickets.

**Describe the bug**

The mission-doc build documentation workflow is failing.

- https://github.com/nasa/cFS/actions/runs/21296014369/job/61301835049

```
Run make -C ./build/docs/mission-doc/latex
make: Entering directory '/home/runner/work/cFS/cFS/build/docs/mission-doc/latex'
rm -f *.ps *.dvi *.aux *.toc *.idx *.ind *.ilg *.log *.out *.brf *.blg *.bbl refman.pdf
pdflatex refman
This is pdfTeX, Version 3.141592653-2.6-1.40.22 (TeX Live 2022/dev/Debian) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
(./refman.tex
LaTeX2e <2021-11-15> patch level 1
L3 programming layer <2022-01-21>
(/usr/share/texlive/texmf-dist/tex/latex/base/article.cls
Document Class: article 2021/10/04 v1.4n Standard LaTeX document class
(/usr/share/texlive/texmf-dist/tex/latex/base/size10.clo))
...
< shortened for reference >
...
LaTeX Warning: Label `cfe__psp__module_8c_a120a92699250549d1cd3aa2eeb67dae0' mu
ltiply defined.

) (/usr/share/texlive/texmf-dist/tex/context/base/mkii/supp-pdf.mkii
[Loading MPS to PDF converter (version 2006.09.02).]
) (/usr/share/texlive/texmf-dist/tex/latex/epstopdf-pkg/epstopdf-base.sty
(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg))
*geometry* driver: auto-detecting
*geometry* detected driver: pdftex
(/usr/share/texlive/texmf-dist/tex/latex/hyperref/nameref.sty
(/usr/share/texlive/texmf-dist/tex/latex/refcount/refcount.sty)
(/usr/share/texlive/texmf-dist/tex/generic/gettitlestring/gettitlestring.sty))
(./refman.out) (./refman.out)
Runaway text?
\@ifundefined {etoctocstyle}{\let \etoc@startlocaltoc \@gobble \let \etoc@setto
cdepth \ETC.
! TeX capacity exceeded, sorry [main memory size=5000000].
<inserted text> ...{\numberline {9.72.2}Funct\ETC.
                                                  
l.146 \begin{document}
                      ^^M
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on refman.log.
make: *** [Makefile:10: refman.pdf] Error 1
make: Leaving directory '/home/runner/work/cFS/cFS/build/docs/mission-doc/latex'
Error: Process completed with exit code 2.
```


**To Reproduce**
Run the standard workflows (on 'main', as of v7.0.0).

**Expected behavior**
A clear and concise description of what you expected to happen.

**Code snips**
If applicable, add references to the software.

**System observed on:**
- GitHub.com Workflows

**Additional context**

- https://github.com/nasa/cFS/issues/875
- https://github.com/nasa/cFS/pull/876

**Reporter Info**
Keegan Moore / NASA GSFC

--- a/.github/workflows/build-documentation.yml
+++ b/.github/workflows/build-documentation.yml
@@ -68,9 +68,10 @@ jobs:
       deploy: false  # Note can't use cache with deploy, deploy in following job instead
 
   build-mission-doc:
+    # see cFS issue #877, here: https://github.com/nasa/cFS/issues/877
+    if: false # This will prevent the job from ever running
     needs: checkout-and-cache
     name: Build Mission Doc
-    #uses: nasa/cFS/.github/workflows/build-deploy-doc.yml
     uses: nasa/cFS/.github/workflows/build-deploy-doc.yml@main
     with:
       target: "[\"mission-doc\"]"

